accelerator/BUILD

```diff
@@ -80,6 +80,21 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "iri_proxy_apis",
+    srcs = ["iri_proxy_apis.c"],
+    hdrs = ["iri_proxy_apis.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":common_core",
```

Comment:
Delete all the `deps` here and use `"@entangled//cclient/api",` to replace them

Reply:
Already removed unnecessary dependencies.